spidermon/contrib/validation/utils.py

```diff
@@ -28,3 +36,43 @@ def is_valid_url(url):
 
 def is_valid_email(email):
     return not EMAIL_REGEX.match(email) is None
+
+
+def get_schema_from(source):
+    if is_schema_url(source):
+        schema = get_contents(source)
+        try:
+            return json.loads(schema)
+        except Exception as e:
+            logger.exception(
+                str(e) + "\nCould not parse schema from '{}'".format(source)
+            )
+    elif source.endswith(".json"):
+        with open(source, "r") as f:
+            try:
+                return json.load(f)
+            except Exception as e:
+                logger.exception(
+                    str(e) + "\nCould not parse schema in '{}'".format(source)
+                )
+    else:
+        schema = load_object(source)
```

Comment:
The documentation does not make it obvious that the schema may be a string defining the import path of a variable defining the schema.

Reply:
Umm, should we give an example of how a schema should be like? I feel that could be done in a more practical way with the tutorial I am building.

Reply:
I agree with you on the purpose of information given in reference and tutorial, but I am not entirely sure about what you mean by `import string` or the review you made. I feel there is a bit of misunderstanding on this. As I don't get how this (_`schema may be a string defining the import path of a variable defining the schema.`_) conclusion is being made.

Reply:
@Gallaecio Can you take another go of what you meant here? It would be better to clear the air by know what exactly you mean by this. 

> If we intend to support an import string as well, I think the documentation above should be expanded accordingly: mention this possibility in the introductory paragraph and feature this possibility in the example.

Reply:
@Gallaecio ping!

Reply:
Oops, I missed this, sorry!

I assume `source` here contains an “import string”, a Python string that indicates the import path of the object to be loaded (e.g. `'myproject.schemas.MY_SCHEMA'`).

Reply:
No worries, @Gallaecio. 

Yes, the source can be an import string as well as the actual object containing the schema. It's up to the load_object method to figure out which is which and return the schema. 

```
def load_object(path):
    """Load an object given its absolute object path, and return it.

    object can be a class, function, variable or an instance.
    path ie: 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware'
    """
```
Hence to better answer the question you asked, I say, yes the documentation doesn't make it obvious and I think we can fix that by adding another alternative approach of adding schemas. Check the `validation.rst` file

Reply:
@Gallaecio Resolving, if this looks good.